src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ExcessHet.java

```diff
@@ -137,11 +137,6 @@ static double exactTest(final int hetCount, final int refCount, final int homCou
         final int rareCopies = 2 * obsHomR + hetCount;
         final int N = hetCount + obsHomC + obsHomR;
 
-        //If the probability distribution has only 1 point, then the mid p-value is .5
```

Comment:
Note that this short circuit for calculating a mid p-value is incorrect, since we are calculating one-sided p-values here. I just went ahead and removed the short circuit.

Reply:
See discussion below about the mid-p correction, but I think it's probably fine to remove the short circuit anyway.